pytorch_toolkit/formula_recognition/tests/test_export.py

```diff
@@ -0,0 +1,76 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import unittest
+
+import numpy as np
+import yaml
+
+from tools.export import ONNXExporter
+
+
+class TestExport(unittest.TestCase):
+    def setUp(self):
+        with open('configs/config.yml', 'r') as f:
+            config = yaml.load(f, Loader=yaml.SafeLoader).get('export')
+        self.config = config
+        self.exporter = ONNXExporter(config)
+
+    def test_1_encoder_export(self):
+        try:
+            self.exporter.export_encoder()
```

Comment:
probably it is better to check existence of exported file rather than absence of thrown exception, is not it?

Reply:
Tests updated